### DISCUSSION TITLE
Provide online interactive example for DTKDeclarative UI library

### DISCUSSION THREAD

**Discussion by ZhongYic00** [linuxdeepin/developer-center#6982] [Features Request & Ideas | 特性请求 & 头脑风暴]
Like in https://zhuzichu520.github.io/ .QML should be easy to publish online, although Qt-wasm is very huge...
At least, a components gallery is necessary.
![image](https://github.com/linuxdeepin/developer-center/assets/38401412/83808ac9-2cd7-4350-b642-1bf4d107dd09)

**Comment by BLumia** [comment]
> .QML should be easy to publish online, although Qt-wasm is very huge...

dtkdeclarative is somewhat bound with DDE's QPA, not sure if it's doable. cc @18202781743 

> At least, a components gallery is necessary.

There is a `dtk-exhibition` binary served for this purpose.

**Reply by 18202781743** [reply]
Now we only have some [examples](https://linuxdeepin.github.io/dtkdeclarative/annotated.html) of documents.
Afterwards, we will continue to optimize `dtk-exhibition`.
